dev-server/documents/html/multi-frames.mustache

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Multi-frame test</title>
+<style>
+  body {
+    font-family: sans-serif;
+  }
+</style>
+</head>
+<body>
+  <h1>Multi-frame test</h1>
+  <p>Selecting text, adding annotations, opening and closing the sidebar here should not have any impact in the other frames.</p>
+  <iframe src="/" width="75%"></iframe>
```

Comment:
I would suggest to use one of the simpler HTML test cases here (eg. the Sherlock Holmes or Robert Burns documents).